### DISCUSSION TITLE
Current version is great. Keep it simple.

### DISCUSSION THREAD

**Discussion by Dim25** [iSoron/uhabits#440] [General Discussion]
Thank you Alinson Xavier (@iSoron) and everybody who have contributed to this app. Amazing job!

There are over 1,000,000+ installs of this app on Google Play, with vast amount of positive reviews. In my opinion the app is in perfect state right now, it's simple to use and extremely efficient. As a user I do not want to see any major updates or new features which can increase the complexity or distract from current core functional, other than necessary bug fixes. So far you are doing a great job in maintaining this state. And I would love to ask you to keep it that way.

To be clear, I'm not against new features per se. It's a vote for proceeding with caution and for keeping in mind the multitude of people who are enjoying the app as is and may not be presented on GitHub to provide their feedback on new feature requests. 

Thank you

_Note: This may be a bit out of format of 'Issues', feel free to delete this or modify in any way._

**Comment by GabrielMPhi** [comment]
Agreed! Simplicity and efficiency is key!
